addons/mail/static/src/models/thread/thread.js

```diff
@@ -658,10 +658,15 @@ function factory(dependencies) {
         /**
          * Marks as read all needaction messages with this thread as origin.
          */
-        async markNeedactionMessagesAsOriginThreadAsRead() {
-            await this.async(() =>
-                this.env.models['mail.message'].markAsRead(this.needactionMessagesAsOriginThread)
-            );
+        async markMessagesAsOriginThreadAsRead() {
```

Comment:
I still disagree with this rename btw, this new name makes the method less understandable than before (we are still marking **needaction** messages as read) and it is not safe to rename in stable.